src/platform/packages/shared/shared-ux/code_editor/impl/code_editor.tsx

```diff
@@ -602,11 +602,12 @@ export const CodeEditor: React.FC<CodeEditorProps> = ({
 
   return (
     <div
-      css={[styles.container, classNameCss]}
+      css={styles.container}
       onKeyDown={onKeyDown}
       data-test-subj={dataTestSubj ?? 'kibanaCodeEditor'}
       className="kibanaCodeEditor"
     >
+      <Global styles={classNameCss} />
```

Comment:
This is moved to be injected globally, since widgets are not rendered outside of the structure of the editor, these style overrides if left as is will not apply anymore to the widgets for consumers